### ISSUE TITLE
Create a better experience in general for development

### ISSUE THREAD

**Issue by tbetous** [microsoft/fast#3711]
**What feature are you requesting?**

It would be nice to have a command that allow to trigger a build when a developper do some modifications in the fast-fundation project.

**What would be the benefit of adding this feature?**

This would be particularly convenient in the case where a storybook is launch in parallel and a developper want to see it's dev on it.

**What solution would be ideal for you?**

I tought to several possibilities : 
- `yarn build:watch`
- `yarn dev:build`

The task seems easy but I didn't found solution with `tsc -w` to do that. This may require an external library such as https://github.com/gilamran/tsc-watch which allows to trigger a command in case of success of `tsc`.

**What alternatives have you considered?**

For now, I have to build every time. It's a little uncomfortable but it works.

**Is there any additional context for your request?**

**Attach screenshots or images add detail to your request:**

**Comment by tbetous** [comment]
I investigated that issue. In my case I'm not sure it is actually needed. When I do `yarn add link:packages/fast-foundation`  in `fast-components` package  and `yarn dev` in `fast-foundation` package, each time I do something in `fast-foundation` while story book is started, `fast-foundation` is rolled up directly in `fast-components` package, so I can see my modifications.

Not sure to understand all the mechanicsm behind that, but it make developping in `fast-foundation` more comfortable. This issue may shoud le be close. I think what is really needed is a small doc to explain how to develop in fast-foundation package.

**Comment by awentzel** [comment]
@tbetous I'm going to repurpose the title to provide documentation strategy for easily testing if that's ok.  Do you want to contribute documentation as you described in your last comment?

**Comment by chrisdholt** [comment]
@awentzel This issue wasn’t unrelated to testing, it was to add a command to help with inner loop development. It seems like this should be a separate issue; I don’t see how this relates to testing in fast foundation.

**Comment by EisenbergEffect** [comment]
We should probably look at creating a second issue for documentation improvements around the inner-loop workflow and then keep this open to see if we can actually create a better experience in general for development.

**Comment by tbetous** [comment]
Hi @awentzel, I would be glad to contribute to fast documentation! 

I think @chrisdholt is right, "testing" is misleading and as @EisenbergEffect said, we sould : 
- Rename this issue with "Create a better experience in general for development", it is very generic but it allows to trace ideas and thoughs about how to improve fast developpement process. 
- Open a new issue "Documentation improvements around the inner-loop workflow of fast-foundation" for my previous comment. It will help future contributors on fast-foundation.

What do you think about that ?

**Comment by EisenbergEffect** [comment]
@tbetous That sounds good to me. Let's go with it. Thanks for being willing to help out!

**Comment by chrisdholt** [comment]
I think this can be closed given #5853 - I will be opening a new issue though as we'll need a visual testing aspect with super baseline styles for ensuring that our templates work as expected in fast-foundation.
